stdlib/os/__init__.pyi

```diff
@@ -722,6 +725,9 @@ def rmdir(path: StrOrBytesPath, *, dir_fd: int | None = ...) -> None: ...
 
 class _ScandirIterator(Iterator[DirEntry[AnyStr]], AbstractContextManager[_ScandirIterator[AnyStr]]):
     def __next__(self) -> DirEntry[AnyStr]: ...
+    def __exit__(
+        self, exc_type: type[BaseException] | None, exc_val: BaseException | None, exc_tb: TracebackType | None
+    ) -> bool | None: ...
```

Comment:
```suggestion
    def __exit__(
        self, *args: object
    ) -> None: ...
```
This is `ScandirIterator_exit` in posixmodule.c. It ignores its args and always returns None.